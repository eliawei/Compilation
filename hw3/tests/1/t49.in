//test

void main()
{
    int x = 4;
    byte y = 100 b;
	if (y == a) print("ok");
}