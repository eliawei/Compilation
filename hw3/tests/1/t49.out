line 7: variable a is not defined
